japicmp-maven-plugin/pom.xml

```diff
@@ -25,45 +25,63 @@
 	</licenses>
 
 	<properties>
-		<maven.version>3.6.0</maven.version>
-		<aether.version>1.1.0</aether.version>
+		<maven.version>3.6.3</maven.version>
```

Comment:
Don't! `${maven.version}` is reserved to an internal maven property providing the currently used Maven version. Maven it `mavenVersion`